subprojects/docs/src/docs/userguide/potentialTraps.adoc

```diff
@@ -25,11 +25,11 @@ For Gradle users it is important to understand how Groovy deals with script vari
 ++++
 <sample xmlns:xi="http://www.w3.org/2001/XInclude" id="scope" dir="userguide/tutorial" title="Variables scope: local and script wide">
             <sourcefile file="scope.groovy"/>
-            <output args=""/>
+            <output args="scope.groovy" executable="groovy" />
```

Comment:
nit, can you reverse executable and args here?

Reply:
done